README.md

```diff
@@ -1 +1,34 @@
-# ticket-booking-app
\ No newline at end of file
+# ticket-booking-app
+The seat reservation system for a multiplex.
+
+## Table of contents
+* [General info](#general-info)
+* [Setup](#setup)
+* [Inspiration](#inspiration)
+* [Technologies](#technologies)
+
+## General info
+The application is created in Scala mainly, however in some case where i couldn't force Spring to work with Scala, I have to use Java features. 
```

Comment:
Czy scala była tu najlepszym rozwiązaniem? może w jakimś innym języku byłoby łatwiej?

Reply:
Scala nie była moim wyborem, była to wytyczna do tego projektu. Tak naprawdę nie wiele wiedziałem o tym języku i chętnie podjąłem się wyzwania nauczenia się go. Napotkałem dwa problemy, które wynikały z użycia Scali, pierwszy z mapowaniem (ORM) kolekcji i typów wyliczeniowych, drugi to zmuszenie Springa do korzystania z mojego settera lub konstruktora w beanie.Co do oceny czy jest dobrym rozwiązaniem do tego typu aplikacji, to jak na razie mam zbyt małe doświadczenia żeby się wypowiedzieć. Nie wiem czy napotkane problemy wynikały z używania Scali czy bardziej z mojej niewiedzy. 